syne_tune/optimizer/schedulers/searchers/sklearn/sklearn_surrogate_searcher.py

```diff
@@ -96,15 +101,14 @@ def __init__(
                 hp_ranges=hp_ranges,
                 estimator=self.estimator,
                 acquisition_class=scoring_class_and_args,
-                skip_local_optimization=True,
-                initial_scoring="acq_func",
                 num_initial_candidates=num_initial_candidates,
                 num_initial_random_choices=num_initial_random_choices,
+                initial_scoring="acq_func",
+                skip_local_optimization=True,
                 allow_duplicates=allow_duplicates,
-                resource_attr=None,  # TODO This needs to be passed for multi-fidelity
```

Comment:
What about resource attribute, shall we necer pass it?

Reply:
What you do in the ICML paper, can easily be implemented with a searcher which does not know about multi-fidelity at all. Namely, we can use a specific `StateForModelConverter`, which converts the multi-fidelity data to single fidelity data before the searcher even sees it.

Reply:
That is why, at least for now, I'd keep it simple here. I am rather a fan of keeping something simple and make it more complex when needed, because at that later point, you know exactly what is needed.

Reply:
I think this is a good rule in general. For now this was just a todo placeholder so I am asking why we do we remove it since nothing changed. I guess it just sets a default so we might not have it here.